Editor/Windows/AssetPaletteWindowHeader.cs

```diff
@@ -207,22 +243,26 @@ private void SetSortModeAndSortCurrentEntries(SortModes sortMode)
 
         private void DoCollectionDropDown(Rect collectionRect)
         {
-            string[] existingCollectionGuids = AssetDatabase.FindAssets($"t:{typeof(AssetPaletteCollection).Name}");
+            
+            string[] existingCollectionGuids = AssetDatabase.FindAssets($"t:{nameof(AssetPaletteCollection)}");
 
             // Allow a new collection to be created.
             GenericMenu dropdownMenu = new GenericMenu();
 
             // Add any existing collections that we find as options.
-            foreach (string collectionGuid in existingCollectionGuids)
+            for (int i = 0; i < existingCollectionGuids.Length; i++)
```

Comment:
What did you change here?

Reply:
Ow, yeah I'm still a foreach hater, I must have changed that one by my code cleanup, want me to revert it?